### PR TITLE
Added the ability to define phase transitions by pressures instead of…

### PR DESCRIPTION
**Pull request by moorek**
… depths.

### CODE REVIEW

**Comment by moorek** [comment]
how do I do this/what would you like me to do?

**Comment by bangerth** [comment]
Nothing for you to do -- she entered the command that triggers a machine to run the tests.

File: run [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# A simple setup for convection in a quarter of a 2d shell. See the
```
Comment (bangerth) [inline run]: You add a file called `run` in the global directory. I don't think that's what you wanted to do, or at least probably not here.

**Comment by jdannberg** [comment]
Great, thanks!

**Comment by bangerth** [comment]
By the way, @moorek , I think this would be worth adding an entry to doc/modules/changes.h. Feel free to create a separate pull request with such an entry!

**Comment by moorek** [comment]
What is that?

On Wed, May 27, 2015 at 3:35 PM, Wolfgang Bangerth <notifications@github.com

**Comment by bangerth** [comment]
Talk to me next time we're in the same room!

**Pull request merged**